sdk/python/pulumi_azure/network/get_public_ips.py

```diff
@@ -60,7 +108,7 @@ def get_public_i_ps(allocation_type=None,attached=None,name_prefix=None,resource
     import pulumi
     import pulumi_azure as azure
 
-    example = azure.network.get_public_i_ps(attached=False,
+    example = azure.network.get_public_ips(attached=False,
```

Comment:
This would also be a breaking change. Working on a codegen fix that special-cases this to use the old name.

Reply:
https://github.com/pulumi/pulumi/pull/5202

Reply:
We'll need to update the `go.mod` require for pulumi/pulumi in this PR to reference this commit: https://github.com/pulumi/pulumi/commit/62925433421370e32e614f2d2647f577673148bc